mesh/silvair-io.c

```diff
@@ -118,6 +120,16 @@ struct silvair_tx_cmd_pld {
 	uint8_t			adv_data[0];
 } __packed;
 
+struct silvair_keep_alive_cmd_pld {
```

Comment:
I'm confused. Is this a command, or event/response? BlueZ doesn't need to send its version to firmware, right?

Reply:
Right. BlueZ uses this struct only for RX purposes.